defaults/main.yml

```
@@ -63,6 +66,9 @@ ethv_validator_keys_folder: "validator_keys"
 # Reboot after update
 ethv_reboot_after_update: yes
 
+# Which client are we going to use? (prysm | lighthouse)
+ethv_client: "prysm"
```

```suggestion
ethv_20_client: "prysm"
```

I'd call it `ethv_20_client` so we can see to which chain it belongs and then move it up to the other `ethv_20_net` variable for visibility